src/app/features/create-event/event-details-form/event-details-form.component.html

```diff
@@ -0,0 +1,138 @@
+        <ng-container *ngIf="errors['required']">To pole jest wymagane.</ng-container>
+      </mat-error>
+    </mat-form-field>
+
+    <mat-form-field color="primary" class="fieldset__input">
+      <mat-label>Kod pocztowy</mat-label>
+      <mat-hint>np. 00-000</mat-hint>
+      <input matInput formControlName="postCode" />
+      <mat-error *ngIf="postCodeCtrl.errors as errors">
+        <ng-container *ngIf="errors['required']">To pole jest wymagane.</ng-container>
+        <ng-container *ngIf="errors['pattern']">Kod pocztowy jest w złym formacie.</ng-container>
+      </mat-error>
+    </mat-form-field>
+
+    <mat-form-field color="primary" class="fieldset__input">
+      <mat-label>Miasto</mat-label>
+      <input matInput formControlName="city" />
+      <mat-error *ngIf="cityCtrl.errors as errors">
+        <ng-container *ngIf="errors['required']">To pole jest wymagane.</ng-container>
+        <ng-container *ngIf="errors['minlength'] as minLength"
+          >Min. liczba znaków: {{ minLength.actualLength }}/{{ minLength.requiredLength }}.</ng-container
+        >
+        <ng-container *ngIf="errors['maxlength'] as maxLength"
+          >Maks. liczba znaków: {{ maxLength.actualLength }}/{{ maxLength.requiredLength }}.</ng-container
+        >
+      </mat-error>
+    </mat-form-field>
+
+    <mat-form-field color="primary" class="fieldset__input">
+      <mat-label>Ulica</mat-label>
+      <input matInput formControlName="street" />
+      <mat-error *ngIf="streetCtrl.errors as errors">
+        <ng-container *ngIf="errors['required']">To pole jest wymagane.</ng-container>
+        <ng-container *ngIf="errors['minlength'] as minLength"
+          >Min. liczba znaków: {{ minLength.actualLength }}/{{ minLength.requiredLength }}.</ng-container
+        >
+        <ng-container *ngIf="errors['maxlength'] as maxLength"
+          >Maks. liczba znaków: {{ maxLength.actualLength }}/{{ maxLength.requiredLength }}.</ng-container
+        >
+      </mat-error>
+    </mat-form-field>
+
+    <mat-form-field color="primary" class="fieldset__input">
+      <mat-label>Numer domu</mat-label>
+      <input matInput formControlName="streetNumber" />
+      <mat-error *ngIf="streetNumberCtrl.errors as errors">
+        <ng-container *ngIf="errors['required']">To pole jest wymagane.</ng-container>
+        <ng-container *ngIf="errors['maxlength'] as maxLength"
+          >Maks. liczba znaków: {{ maxLength.actualLength }}/{{ maxLength.requiredLength }}.</ng-container
+        >
+      </mat-error>
+    </mat-form-field>
+
+    <mat-form-field color="primary" class="fieldset__input">
+      <mat-label>Lokal</mat-label>
+      <input matInput formControlName="apartmentNumber" />
+      <mat-error *ngIf="apartmentNumberCtrl.errors as errors">
+        <ng-container *ngIf="errors['maxlength'] as maxLength"
+          >Maks. liczba znaków: {{ maxLength.actualLength }}/{{ maxLength.requiredLength }}.</ng-container
```

Comment:
możemy dodać takie sprytne liczniki przy polach textarea gdzie będziemy pokazać aktualną/maksymalną liczbę wpisanych znaków, oczywiście to feature także nowy task do product backlogu